.docker/entrypoint.sh

```diff
@@ -28,7 +28,7 @@ export PYSPARK_DRIVER_PYTHON=/opt/conda/bin/python
 # python coverage-threshold install
 # pip install coverage-threshold
 
-coverage run --branch -m pytest --junitxml=pytest-results.xml -n 2 .
```

Comment:
I'm not completely sure, but doesn't the `-n 2` specify the number of processes, is there a reason this is removed? It is fine by me just curious. 😄

Reply:
That is correct. ;)
We only have 2 tests for now, so it isn't needed.
I plan to split tests for `telemetry` and `spark_sql_migrations` in the pipeline. Mandalorian has done something similar.